pilot/pkg/kube/inject/inject.go

```diff
@@ -316,8 +315,8 @@ func injectRequired(ignored []string, namespacePolicy InjectionPolicy, podSpec *
 	return required
 }
 
-func formatDuration(in *duration.Duration) string {
-	dur, err := ptypes.Duration(in)
+func formatDuration(in *types.Duration) string {
```

Comment:
same with the other comments above - I don't mind the change, cleanups and refactorings - but in separate PRs.